corehq/apps/reports/templates/reports/form/edit_submission.html

```diff
@@ -2,7 +2,7 @@
 {% load hq_shared_tags %}
 {% load i18n %}
 
-{% block head %}{{ block.super }}
+{% block js %}{{ block.super }}
```

Comment:
See the note [here](https://github.com/dimagi/commcare-hq/pull/9628/files#diff-ba8e2053fc2f55f1466a0077565c13c6R4)